build-tests-subspace/rush-lib-test/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "rush-lib-test",
+  "version": "0.0.0",
+  "private": true,
+  "description": "A minimal example project that imports APIs from @rushstack/rush-lib",
+  "license": "MIT",
+  "scripts": {
+    "build": "rimraf ./lib/ && tsc",
```

Comment:
Use Heft?

Reply:
Yeah, I am going to use Heft. The `tsc` could not pass the CI either.